lib/src/git.rs

```diff
@@ -313,18 +313,24 @@ pub fn import_some_refs(
         }
     }
 
-    // Find commits that are no longer referenced in the git repo and abandon them
-    // in jj as well.
+    let abandoned_commits = abandon_unreachable_commits(changed_remote_refs, mut_repo);
+    let stats = GitImportStats { abandoned_commits };
+    Ok(stats)
+}
+
+/// Finds commits that used to be reachable in git that no longer are reachable.
+/// Those commits will be recorded as abandoned in the `MutableRepo`.
+fn abandon_unreachable_commits(
+    changed_remote_refs: BTreeMap<RefName, (RemoteRef, RefTarget)>,
+    mut_repo: &mut MutableRepo,
+) -> Vec<CommitId> {
```

Comment:
nit: `mut_repo` is usually the first argument, and `changed_remote_refs` can be borrowed.

Reply:
Done. (I let VS Code extract the function and I forgot to check the generated signature.)